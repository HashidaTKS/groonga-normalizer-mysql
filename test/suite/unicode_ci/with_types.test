register normalizers/mysql

normalize NormalizerMySQLUnicodeCI "a1!あア亜💕 " WITH_TYPES
