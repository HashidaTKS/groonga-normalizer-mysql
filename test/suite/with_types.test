register normalizers/mysql

normalize NormalizerMySQLGeneralCI "a1!あア亜💕 " WITH_TYPES
