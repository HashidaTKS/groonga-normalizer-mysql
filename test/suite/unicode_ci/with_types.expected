register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLUnicodeCI "a1!あア亜💕 " WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A1!ああ亜💕 ",
    "types": [
      "alpha",
      "digit",
      "symbol",
      "hiragana",
      "hiragana",
      "kanji",
      "others",
      "others"
    ]
  }
]
